implementations/rust/ockam/ockam_transport_tcp/Cargo.toml

```diff
@@ -32,6 +32,7 @@ ring = ["tokio-rustls/ring"]
 
 [dependencies]
 cfg-if = "1.0.0"
+log = "0.4"
```

Comment:
Was this accidentally added?  [this](https://github.com/build-trust/ockam/blob/15620309f4ff93326e59decf23801e9ecfed348b/implementations/rust/ockam/ockam_transport_tcp/src/portal/interceptor.rs#L2) should use tracing